### ISSUE TITLE
share Obsidian know-how with the TTW community

### ISSUE THREAD

**Issue by ragamouf** [the-turing-way/the-turing-way#3560]
### Summary

_Background_

This came up in a thread on the TTW slack in response to a call out on time management techniques. 
[Obsidian](https://obsidian.md/) is a free and open source note-taking tool (but so much more) that is built locally on a network of markdown files. Some people use it for project management, others for note-taking. I heard of a uni administration using it to guide their re-registration process!

Could people across this community pool their knowledge, advice, there-be-dragons warnings etc together and share it somehow? This issue could build as a thread (channel even?!) in the slack, turn into a session in one of the community calls, end up as a chapter in project design?

### What needs to be done?

- [ ] Start an issue to test if this idea resonates
- [ ] Apply the saying no flowchart as we scope the idea
- [ ] Interrogate this "we" business - who's on board?
- [ ] gather tips, tricks, resources
- [ ] assemble into a communicable manifestation
- [ ] agree on start and finish dates
- [ ] anything else I've not considered!

### Who can help?

I'd like to tag people who've mentioned Obsidian in the TTW slack - eg @stefanocoretta @aleesteele @kallewesterling @ninadicara @malvikasharan @Arielle-Bennett

honestly, anyone who has success and failures to share, or is intrigued to learn how this tool might be relevant is welcome.
---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by stefanocoretta** [comment]
I guess I can start off with what I like the most about Obsidian:

- It's free.
- Hierarchical dependencies between notes are dealt with by links, rather than location of the note.
- You write in Markdown.
- It's both a desktop and mobile app.
- You can link your bibliography files and insert citations in notes.

**Comment by ninadicara** [comment]
To add to that list: 
- you can track files/vaults with git, since they're just markdown, which I really like.

A 'gotcha' I've noticed - it doesn't play super well with cloud drives like OneDrive - the files sometimes end up in weird places. 
I didn't know about the bibliography files @stefanocoretta 👀

**Comment by aleesteele** [comment]
@stefanocoretta, @ninadicara, and @ragamouf - this might be a great topic to write about at Book Dash this June! 😉

**Comment by ragamouf** [comment]
I was listening to the [On The Reg podcast](https://onthereg.buzzsprout.com/1187141/14790865-obsidian-1996-called-and-it-wants-its-database-back
): the ones who really encouraged me to fire up obsidian and start using it.

Also, here's a [kindly guide](https://thesiswhisperer.com/2023/02/01/building-a-second-brain-for-writing-with-obsidian/) from The Thesis Whisperer (one of the above podcast hosts) on using Obsidian for writing.

**Comment by ninadicara** [comment]
I do use GitHub to back up yes ! Unhealthy addiction to using version control, and I like being able to see it build over time, and not needing to worry about deleting things 😄 Gutted I'm not able to make the June Bookdash dates but would be happy to contribute outside of that anyway. 
I wonder if there is a bigger theme here that Obsidian fits into and would be a case study of, along with other second-brain type software - my only worry about doing a specific chapter on Obsidian in a guide like the TW is that it could go out of date and need updating as features change etc. But perhaps case studies of how people organise their academic work using tools like obsidian (also thinking of tiddlywiki as a similar one, sure there are others as in the Slack thread) could be really useful to people as a jumping off point if they are looking for inspiration on new tools?

**Comment by Arielle-Bennett** [comment]
Oh @ninadicara I love the idea of a "second brain" chapter, and subchapters on use cases with different systems...

**Comment by stefanocoretta** [comment]
I use iCloud to sync across devices rather than GitHub (I don't care much about version tracking of notes and I am lazy 😄). Where can I find info about the June Bookdash?

**Comment by ragamouf** [comment]
@stefanocoretta https://theturingway.slack.com/archives/C014KE9A9LM/p1710526257347769

**Comment by stefanocoretta** [comment]
Hi! Sorry this time of the year is quite hectic for me! I think I'll have time to look into everything later this summer

**Comment by ragamouf** [comment]
Yeah, I think this is a slow burn for many of us. (Now that I've realised we missed the book dash deadline!)
How about we return to this at the beginning of August? I'll take thumbs up as a yes. In the meantime, perhaps we can reflect on how we use obsidian in terms of specific areas of work or tasks to achieve. That would help us shift away from 'how to use this tool Obsidian' towards 'How I use [tool x] for [task y]'.  

- for academic writing with citation plugins
- as a productivity tool
- managing workflows
- collaboration
- to practice version control

**Comment by ninadicara** [comment]
@ragamouf Thanks that sounds like a plan - also struggling with keeping up this time of year!

**Comment by ragamouf** [comment]
Aaand, just like that, it's August, and I'm really stretching the definition of "beginning", swinging in on the 12th!
Indeed I have fallen from the darkly shining tower of Obsidian, I barely use it, so I'm not as invested as I was in this topic in February, when it seemed like at the beginning of the year there was still promise to learn and practice a new system.
How are folks feeling about continuing this issue? It's really ok to wrap it up and close out. There's definitely opportunity for someone to take it and run with it, but I don't think I have capacity to lead it.
Tagging @stefanocoretta @ninadicara @aleesteele @Arielle-Bennett
